Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Dockerfile for Pokemon Showdown Server
+FROM node:10
```

Comment:
We're about to upgrade to requiring Node 13, btw

Reply:
I can swap this to Node 13.